cmd/k8s-operator/ingress.go

```diff
@@ -220,6 +220,18 @@ func (a *IngressReconciler) maybeProvision(ctx context.Context, logger *zap.Suga
 			},
 		},
 	}
+	for _, ip := range tsIPs {
```

Comment:
I'm not sure that means we shouldn't add the information, the hostname _does_ resolve to this IP. But I'll remove it

Reply:
yeah the hostname resolves to this IP, but you can't connect to it directly unlike the service case.
e.g. if you try to `curl http://<ingress-ip>` won't work, and `https://<ingress-ip>` will return an invalid cert.

Reply:
Yeah but `curl -k` should work. I just mean I believe there's nothing in the ingress API contract that says TLS certs have to contain the address as a SAN.

Reply:
actually it doesn't even return an invalid cert because it does matching based on SNI
e.g. 
```
➜  ~ curl https://100.125.4.132
curl: (35) LibreSSL/3.3.6: error:1404B438:SSL routines:ST_CONNECT:tlsv1 alert internal error
```

as opposed to 
```
➜  ~ curl https://nginx.tail-scale.ts.net
<!DOCTYPE html>
<html>
<head>
<title>Welcome to nginx!</title>
<style>
html { color-scheme: light dark; }
body { width: 35em; margin: 0 auto;
font-family: Tahoma, Verdana, Arial, sans-serif; }
</style>
</head>
<body>
<h1>Welcome to nginx!</h1>
<p>If you see this page, the nginx web server is successfully installed and
working. Further configuration is required.</p>

<p>For online documentation and support please refer to
<a href="http://nginx.org/">nginx.org</a>.<br/>
Commercial support is available at
<a href="http://nginx.com/">nginx.com</a>.</p>

<p><em>Thank you for using nginx.</em></p>
</body>
</html>
```

Reply:
Is that the case even without funnel? Hmm ok, yeah then it should not be there

Reply:
> Is that the case even without funnel?

yep